### ISSUE TITLE
Japanese text after winning Tales of Dragon Buster

### ISSUE THREAD

**Issue by Kilicool64** [lifebottle/Tales-of-the-Abyss-Undub#5]
I just won a game of Tales of Dragon Buster and got a message about one of the game's Easter eggs, but it's in Japanese.
![Tales of the Abyss_SLUS-21386_20230429165639](https://user-images.githubusercontent.com/53983097/235309332-0c7a7e87-57cb-4c14-8fe5-03b66835c8f0.png)

**Comment by swosho** [comment]
All right, this one was actually in the vanilla NA version of the game, unrelated to Japanese ToDB instructions text mistakenly present in the old undub. That said, this oddity is fixed in v2.6 now, thanks.

**Comment by Kilicool64** [comment]
Yeah, I learned about that as well, but it seems I only commented about it on Discord. Glad to see it's fixed now anyway.
